tasks/42_first-n-smallest/first-n-smallest.js

```diff
@@ -0,0 +1,18 @@
+export function firstNSmallest(arr, n) {
+  if (n === 0) return [];
+
+  let nSmallest = [];
+  for (let i = 0; i < n; i++) {
+    nSmallest.push(arr[i]);
+  }
+  let maxInNSmallest = Math.max(...nSmallest);
```

Comment:
```suggestion
  const maxInNSmallest = getMaxInNSmallest(arr)
```
Extract logic for getting maxInNSmallest into separate function to reduce complexity.